debian/postrm

```diff
@@ -27,6 +27,13 @@ case "$1" in
         # remove ESM keyring files created by enable-esm
         rm -f /etc/apt/trusted.gpg.d/ubuntu-esm-keyring.gpg
         rm -f /etc/apt/trusted.gpg.d/ubuntu-esm-v2-keyring.gpg
+        # ESM sources.list.d snippet
+        series=$(grep ^DISTRIB_CODENAME= /etc/lsb-release | cut -d = -f 2)
+        if [ -n "${series}" ]; then
+            rm -f /etc/apt/sources.list.d/ubuntu-esm-${series}.list
```

Comment:
Should we also remove or reinstate /etc/apt/sources.list.d/ubuntu-esm-${series}.list.save if present?

Reply:
I wonder. The point of keeping a save file around is because previously the creds would be in this file. But now they are not, so keeping this file around is a bit useless in my opinion, and even generating it is a bit useless. In the spirit of doing the least amount of changes possible, I kept it, that's all.

Reply:
I think I'll remove it, since I'm also removing the auth.conf.d snippet

Reply:
Although... even when removing ppas, a .save file is kept